# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../duma/no_test.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-07-15 07:15:44.000000000 -0400
+++ ./Makefile	2007-07-15 12:52:28.000000000 -0400
@@ -170,19 +170,6 @@
 SO_OBJECTS = dumapp_so.o duma_so.o sem_inc_so.o print_so.o
 
 all:	libduma.a tstheap$(EXEPOSTFIX) dumatest$(EXEPOSTFIX) dumatestpp$(EXEPOSTFIX) testoperators$(EXEPOSTFIX) $(DUMA_DYN_DEPS)
-	@ echo
-	@ echo "Testing DUMA (static library):"
-	$(CURPATH)dumatest$(EXEPOSTFIX)
-	$(CURPATH)tstheap$(EXEPOSTFIX) 3072
-	$(CURPATH)testoperators$(EXEPOSTFIX)
-	@ echo
-	@ echo "DUMA static confidence test PASSED."
-	@ echo
-	@ echo "Testing DUMA (dynamic library)."
-	(export LD_PRELOAD=./$(DUMASO); exec $(CURPATH)tstheap_so 3072)
-	@ echo
-	@ echo "DUMA dynamic confidence test PASSED."
-	@ echo
 
 install: libduma.a duma.3 $(DUMASO)
 	$(INSTALL) -m 755 duma.sh $(BIN_INSTALL_DIR)/duma
